Plugins/Race/README.md

```diff
@@ -125,18 +125,19 @@ NWNX_Race_SetRacialModifier(150, NWNX_RACE_MODIFIER_ABVSRACE, RACIAL_TYPE_DRAGON
 The `AC` entry is used to modify the base AC for a race.
 
```

Comment:
A note here that non-dodge AC types won't stack with other non-dodge AC from spells effects, would be good

Reply:
Done